UI/GameSettingsScreen.cpp

```diff
@@ -290,6 +290,34 @@ void GameSettingsScreen::CreateViews() {
 		return g_Config.iRenderingMode != FB_NON_BUFFERED_MODE;
 	});
 
+	if (g_Config.sPostShaderSettingName1 != "")  {
```

Comment:
Personally, I would use PostShader.h and get these straight from there.  i.e... (syntax not tested):

```c++
const ShaderInfo *shaderInfo = GetPostShaderInfo(g_Config.sPostShaderName);

if (shaderInfo && !shaderInfo->settingName1.empty()) {
	auto &value = g_Config.mPostShaderSetting.insert(std::make_pair(g_Config.sPostShaderName + "SettingValue1", shaderInfo->settingValue1))->first->second;
	graphicsSettings->Add(new PopupSliderChoiceFloat(&value, shaderInfo->minSettingValue1, shaderInfo->maxSettingValue1, shaderInfo->settingName1, shaderInfo->settingStep1, screenManager()));
}
```

The way it's currently working, it's kinda using g_Config as "global variables" to cache these things.  But on Windows (and maybe Qt?), that may cause confusion because you can actually change the shader from the menu.  If this gets it directly, that's less likely to get out of date.

-[Unknown]